src/components/gltf-model-plus.js

```diff
@@ -31,57 +31,54 @@ AFRAME.GLTFModelPlus = {
   }
 };
 
-// From https://gist.github.com/cdata/f2d7a6ccdec071839bc1954c32595e87
-// Tracking glTF cloning here: https://github.com/mrdoob/three.js/issues/11573
-function cloneGltf(gltf) {
-  const skinnedMeshes = {};
-  gltf.scene.traverse(node => {
-    if (!node.name) {
-      node.name = node.uuid;
-    }
-    if (node.isSkinnedMesh) {
-      skinnedMeshes[node.name] = node;
-    }
-  });
+function parallelTraverse(a, b, callback) {
+  callback(a, b);
 
-  const clone = {
-    animations: gltf.animations,
-    scene: gltf.scene.clone(true)
-  };
+  for (let i = 0; i < a.children.length; i++) {
+    parallelTraverse(a.children[i], b.children[i], callback);
+  }
+}
 
-  const cloneBones = {};
-  const cloneSkinnedMeshes = {};
+function cloneSkinnedMesh(source) {
+  const cloneLookup = new Map();
```

Comment:
just a style thing, but maps like this I lke to name FooToBar so you know what the key/value space is. so sourceToCloneNode etc